### ISSUE TITLE
Sound through VNC

### ISSUE THREAD

**Issue by KleinByte** [TigerVNC/tigervnc#1211]
**Is your feature request related to a problem? Please describe.**
In the evermore remote working increase, it might be a good idea to implement sound over vnc. 
In my situation, I have a surface device from msft that can't boot linux. I use tigervnc to connect to my linux server at home. It would be nice if I could hear sound. 

**Describe the solution you'd like**
TigerVNC sends sound over along with display.

**Describe alternatives you've considered**
There are other solutions that have implemented this like realVNC. But I'd much rather support this team and their open source work, (Pretty sure fedora guys work on this?)

**Additional context**
I dont know how complex the solution would be, I'd assume the hard work is in creating a virtual audio driver for the os to recognize. Not sure. Thanks Guys.

**Comment by breisig** [comment]
This would be a GREAT feature.

**Comment by CendioOssman** [comment]
Client side support for QEMU's audio extension is happening in #1478.

**Comment by lmirguet** [comment]
I'm also hoping to see this feature one day! Thank you to all the contributors!

**Comment by notsilverhoft** [comment]
2 years later, has this been implemented? I'm assuming no, as the issue is still open, but I thought I'd ask

**Comment by CendioOssman** [comment]
No, I'm afraid there hasn't been any progress on this. #1478 is also awaiting more attention, so if someone has time to test and review, then that can help things along.

**Comment by notsilverhoft** [comment]
Ok, thanks, I could check it out, I'm more of a Linux user, but I can still look through it.

**Comment by johntdavis84** [comment]
Hello! I'm curious about the status of this feature request.

TigerVNC just got support for GPU acceleration through Intel's 12th and 13th gen iGPU (via the `ntel-media-va` library). This makes it one of the few Remote Desktop solutions that can use these iGPUs: the only other one I know is a special development testing branch of `xRDP`. (Neither Sunshine nor NoMachine can take advantage of the GPU for hardware acceleration).

If TigerVNC had sound support, it'd be an alternative to the Intel iGPU enabled `xRDP`, which is only available as a package for Ubuntu-based systems.

**Comment by BikerFreak86** [comment]
Oh yes, I'ld also love this feature!

**Comment by LanceJZ** [comment]
Why is there no audio?

**Comment by CendioOssman** [comment]
#2138 has now been merged. This gives audio support in the client when the server is QEMU. It's not a complete solution, but it will resolve some cases.
